source/_integrations/mqtt.markdown

```diff
@@ -559,44 +559,233 @@ The following software has built-in support for MQTT discovery:
 
 ### Discovery examples
 
-#### Motion detection (binary sensor)
+#### Creating a single sensor entity (motion detection binary sensor)
 
-A motion detection device which can be represented by a [binary sensor](/integrations/binary_sensor.mqtt/) for your garden would send its configuration as JSON payload to the Configuration topic. After the first message to `config`, then the MQTT messages sent to the state topic will update the state in Home Assistant.
+In the following example, a single entity will be created representing the state of a motion detection sensor. The motion detection sensor has only 2 states, therefore it can be represented by an [MQTT Binary Sensor](/integrations/binary_sensor.mqtt/).
```

Comment:
I am not happy with changes like `single entity`. Further it is not needed to say a binary sensor has only 2 states

Reply:
Would you prefer the title to say "Creating a sensor entity (motion detection binary sensor)" and for the following text to say "In the following example, an entity will be created representing the state of a motion detection sensor. The motion detection sensor can be represented by an [MQTT Binary Sensor](/integrations/binary_sensor.mqtt/)."?